consensus/fork_choice/src/fork_choice.rs

```diff
@@ -644,6 +664,8 @@ where
         state: &BeaconState<E>,
         payload_verification_status: PayloadVerificationStatus,
         spec: &ChainSpec,
+        progressive_balances_mode: ProgressiveBalancesMode,
```

Comment:
Very minor, but it might be nice to order these `progressive_balances_mode`, `spec`, `logger`. We usually keep `spec` and `logger` at the end.

Reply:
Updated in https://github.com/sigp/lighthouse/commit/5fb3d09a9cfc7a9757638f1388d21337fc093699